Library/Homebrew/utils.rb

```diff
@@ -188,14 +188,32 @@ def self.homebrew_version_string
 
   def self.install_gem_setup_path!(gem, version = nil, executable = gem)
     require "rubygems"
-    ENV["PATH"] = "#{Gem.user_dir}/bin:#{ENV["PATH"]}"
 
-    args = [gem]
-    args << "-v" << version if version
+    if Gem::Specification.find_all_by_name(gem, version).empty?
+      ohai "Installing missing or updating outdated '#{gem}' gem"
```

Comment:
> I think it reads better as just `"Installing or updating '#{gem}' gem"`.

Agreed.

> This is gonna be a lot more noisy if it's printed for every gem.

I found it genuinely useful to see why my invocation of `brew <command>` is stalling before the first output from `gem install` hits my terminal. And it is only one line for the originally requested gem, not for every dependency that results from that. (See the longer example output from my original comment. The ensuing output of `gem install` is much noisier.) Finally, the worst case scenario is that we get three additional lines of output, because there are only three places in core that call this method and request a gem (commands `man`, `style`, and `tests` requesting one of `ronn`, `rubocop`, and `bundler`). Does that sound convincing?